root/components/Tabs.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+// Copyright (C) 2017 MetaBrainz Foundation
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+
+type LinkT = [string, React.Node];
+
+type Props = {
+  list: LinkT[];
+  page: string;
+};
+
+const Tabs = ({list, page}: Props) => {
+  const tabs =  list.map((link) => {
+    return <li className={link[0] == page ? "sel" : null} key={link[0]}>{link[1]}</li>;
```

Comment:
I think I would use a different, simpler approach that doesn't involve building lists and mapping them here.

1. Make the `Tabs` component accept no props. Just embed `{children}` and expect consumers to pass in the `<li>`s.
2. Replace `buildLinkObject` with an `EntityTabLink` component:
   ```JavaScript
   const EntityTabLink = ({selected, ...linkProps}) => (
     <li className={selected ? 'sel' : null}>
       <EntityLink {...linkProps} />
     </li>
   );
   ```
3. In the `EntityTabs` component, pass `page` to `buildLinks`. You can use this to set the `selected` prop on each `EntityTabLink` component.